stats/statsview.cpp

```diff
@@ -719,13 +719,17 @@ void StatsView::QuartileMarker::updatePosition()
 		      x + quartileMarkerSize / 2.0, y);
 }
 
-StatsView::RegressionLine::RegressionLine(double a, double b, QPen pen, QtCharts::QChart *chart, StatsAxis *xAxis, StatsAxis *yAxis) :
-	item(new QGraphicsLineItem(chart)),
+StatsView::RegressionLine::RegressionLine(double a, double b, double width, QBrush brush, QtCharts::QChart *chart, StatsAxis *xAxis, StatsAxis *yAxis) :
+	item(new QGraphicsPolygonItem(chart)),
 	xAxis(xAxis), yAxis(yAxis),
-	a(a), b(b)
+	a(a), b(b), width(width)
 {
-	item->setZValue(ZValues::chartFeatures);
+	item->setZValue(ZValues::chartFeatures - 0.5);
+	QColor invisible(Qt::white);
+	invisible.setAlpha(0);
+	QPen pen(invisible);
 	item->setPen(pen);
```

Comment:
I think you can replace the previous four lines by
```
item->setPen(Qt::NoPen);
```
At the very least you can replace the color thing by `Qt::transparent`.